imgtool/README.rst

```diff
@@ -1,6 +1,29 @@
 Image Manipulation Tool
 #######################
 
+.. note:: Special installation instructions:
+   Due to some configuration issues with servers, it may not be
+   possible to build imgtool with a simple ``go get`` command.  It is
+   possible to build the tool with the following steps::
+
+       # Set GOPATH to a work dir
+       $ mkdir -p $GOPATH/src/mynewt.apache.org
+       $ mkdir -p $GOPATH/src/github.com
+       $ cd $GOPATH/src/mynewt.apache.org
+       $ git clone -b develop https://github.com/apache/incubator-mynewt-newt newt
+       $ cd $GOPATH/src/github.com
+       $ git clone https://github.com/runtimeco/mcuboot
```

Comment:
I appear to have left the last step out of the instructions. There should be one more step, which is the go get from below.